Cargo.toml

```diff
@@ -5,5 +5,5 @@ authors = ["Will S. Medrano <will.s.medrano@gmail.com>"]
 
 [dependencies]
 bitflags = "0.7.0"
-jack-sys = "0.1.0"
```

Comment:
Inb4 you ask about this-- I have a PR to `jack-sys` that will make the changes I need. See https://github.com/Boddlnagg/jack-sys/pull/1.